src/controls/Control.ts

```diff
@@ -32,6 +33,13 @@ export interface ControlProps {
     id: string;
 }
 
+export interface APLComponentProps {
+    /**
+     * Defines the render style of APL component produced by the control.
+     */
+    renderStyle: any;
```

Comment:
type = string?  or do we need the total flexibility of `any`

Reply:
No, I think type: string is better than any.
Fixed 👍